qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -10,6 +10,9 @@
     image_size_stg0 = 1G
     remove_image_stg0 = yes
     force_create_image_stg0 = yes
+    max_block_size = 2199023255552
+    ovmf:
+        max_block_size = 17592186044416
```

Comment:
Please move this part to the variants: `- with_block_resize:`.

Reply:
Changed,many thanks.